git-multimail/git_multimail.py

```diff
@@ -803,7 +803,12 @@ def _compute_values(self):
         dictionary."""
 
         values = self.environment.get_values()
-        fromaddr = self.environment.get_fromaddr(change=self)
+
+        # multimailhook.from is a global override, superceding any
+        # environment-specific mechanisms.
+        config = Config('multimailhook')
+        fromaddr = config.get('from') or \
+            self.environment.get_fromaddr(change=self)
```

Comment:
Yes, it's dead code. But this patch was intended as an RFC to show my idea to receive your feedback. If you don't reject it at this stage, I'll proceed to remove  multimailhook.from handling from other places.